di-ipv-credential-issuer-stub/deploy/face-to-face/template.yaml

```diff
@@ -60,6 +60,10 @@ Parameters:
     Description: "Environment Variables for ECS - Retrieved from SSM Parameter Store"
     Type: AWS::SSM::Parameter::Value<String>
     Default: "/stubs/credential-issuer/f2f/env/F2F_STUB_QUEUE_URL" #pragma: allowlist secret
+  CredentialMitigationEnabled:
+    Description: "Environment Variables for ECS - Retrieved from SSM Parameter Store"
+    Type: AWS::SSM::Parameter::Value<String>
+    Default: "/stubs/credential-issuer/f2f/env/MITIGATION_ENABLED" #pragma: allowlist secret
```

Comment:
Do we need to supply the actual configuration for this somewhere, or is it done manually?

Reply:
That'll be - https://github.com/govuk-one-login/ipv-stubs-common-infra/pull/134